Source/WebCore/platform/CaretAnimator.h

```diff
@@ -29,23 +29,25 @@
 #include "ReducedResolutionSeconds.h"
 #include "Timer.h"
 
+#if PLATFORM(MAC)
+#include "CaretAnimatorType.h"
+#endif
+
 namespace WebCore {
 
 class CaretAnimator;
+class Color;
 class Document;
+class FloatRect;
+class GraphicsContext;
+class Node;
 class Page;
 
-enum class CaretAnimatorType {
-    Default,
-    Alternate
-};
```

Comment:
Is there a reason we're moving the enum out of here?

Reply:
It is to avoid including CaretAnimator.h in WebPage.cpp and WebPageProxy.cpp

But maybe I can use a forward declaration of the enum class instead